client/galaxy/style/scss/theme/blue.scss

```diff
@@ -19,7 +19,7 @@ $brand-masthead-text-hover: $brand-toggle;
 $text-color: $brand-dark;
 $text-muted: lighten($text-color, 10%);
 $text-light: lighten($text-color, 20%);
-$border-color: lighten($text-color, 60%);
+$border-color: $brand-secondary;
```

Comment:
Thank you for working on this. I think removing the borders from the tables is a great idea but I would keep the borders for buttons. I think it improves their contrast. Either way remaining borders should be derived from text colors such that 'brand-colors' can be picked at random without affecting the contrast and legibility of texts and their borders. There is more information on this available at #8388.

Reply:
In my humble opinion taking away the borders gives it more an app/software feeling instead of a bootstrap website feeling, and more modern. But this is off course a personal opinion ;)